http/cookie.ts

```diff
@@ -19,3 +83,52 @@ export function getCookie(rq: ServerRequest): Cookie {
   }
   return {};
 }
+
+/**
+ * Set the cookie header properly in the Response
+ * @param res Server Response
+ * @param cookie Cookie to set
+ * @param [cookie.name] Name of the cookie
+ * @param [cookie.value] Value of the cookie
+ * @param [cookie.expires] Expiration Date of the cookie
+ * @param [cookie.maxAge] Max-Age of the Cookie. Must be integer superior to 0
+ * @param [cookie.domain] Specifies those hosts to which the cookie will be sent
+ * @param [cookie.path] Indicates a URL path that must exist in the request.
+ * @param [cookie.secure] Indicates if the cookie is made using SSL & HTTPS.
+ * @param [cookie.httpOnly] Indicates that cookie is not accessible via Javascript
+ * @param [cookie.sameSite] Allows servers to assert that a cookie ought not to be
+ *  sent along with cross-site requests
+ * Example:
+ *
+ *     setCookie(response, { name: 'deno', value: 'runtime',
+ *        httpOnly: true, secure: true, maxAge: 2, domain: "deno.land" });
+ */
+export function setCookie(res: Response, cookie: Cookie): void {
+  if (!res.headers) {
+    res.headers = new Headers();
+  }
+  // TODO (zekth) : Add proper parsing of Set-Cookie headers
+  // Parsing cookie headers to make consistent set-cookie header
+  // ref: https://tools.ietf.org/html/rfc6265#section-4.1.1
+  res.headers.set("Set-Cookie", cookieStringFormat(cookie));
+}
+
+/**
+ *  Set the cookie header properly in the Response to delete it
+ * @param res Server Response
+ * @param CookieName Name of the cookie to Delete
+ * Example:
+ *
+ *     delCookie(res,'foo');
+ */
+export function delCookie(res: Response, CookieName: string): void {
+  if (!res.headers) {
+    res.headers = new Headers();
+  }
+  const c: Cookie = {
+    name: CookieName,
+    value: "",
+    expires: new Date(0)
+  };
+  res.headers.set("Set-Cookie", cookieStringFormat(c));
```

Comment:
To delete a cookie you have to set the expire time to earlier. Like golang does is a fallback. Looking at rfc the max age under 0 is invalid

Reply:
Ok but I think `delCookie` should be a thin wrapper around `setCookie`, because code like this
```
  if (!res.headers) {
    res.headers = new Headers();
  }
```
is repeated.

Sorry for so many nitpicks :)

Reply:
no my bad!